tests/components/ezviz/test_config_flow.py

```diff
@@ -64,17 +72,51 @@ async def test_user_form_unexpected_exception(hass, ezviz_config_flow):
     assert result["reason"] == "unknown"
 
 
-async def test_user_form_single_instance_allowed(hass, ezviz_config_flow):
-    """Test that configuring more than one instance is rejected."""
+async def test_async_step_import(hass, ezviz_config_flow):
+    """Test the config import flow."""
+    await async_setup_component(hass, "persistent_notification", {})
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_IMPORT}, data=YAML_CONFIG
+    )
+    assert result["type"] == RESULT_TYPE_CREATE_ENTRY
+
+
+async def test_async_step_import_camera(hass, ezviz_config_flow):
+    """Test the config import camera flow."""
+    await async_setup_component(hass, "persistent_notification", {})
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_IMPORT}, data=YAML_CONFIG_CAMERA
+    )
+    assert result["type"] == RESULT_TYPE_CREATE_ENTRY
+    assert result["data"] == {
+        "password": "test-password",
+        "serial": "C666666",
+        "username": "test-username",
+    }
+
+
+async def test_async_step_import_abort(hass, ezviz_config_flow):
+    """Test the config import flow with invalid data."""
+    await async_setup_component(hass, "persistent_notification", {})
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_IMPORT}, data=YAML_INVALID
+    )
+    assert result["type"] == RESULT_TYPE_ABORT
+
+
+async def test_user_form_2nd_instance_returns_form(hass, ezviz_config_flow):
+    """Test that configuring 2nd instance returns form."""
     await init_integration(hass, skip_entry_setup=True)
 
     result = await hass.config_entries.flow.async_init(
         DOMAIN,
         context={"source": SOURCE_USER},
         data=USER_INPUT,
     )
-    assert result["type"] == RESULT_TYPE_ABORT
-    assert result["reason"] == "single_instance_allowed"
+    assert result["type"] == RESULT_TYPE_FORM
```

Comment:
Please complete the flow in each test to either an error, abort or create entry result.

Reply:
Done. Added a mock request to end method with "create entry".